Snippets/JWTKitExamples.swift

```diff
@@ -63,14 +61,13 @@ do
     // snippet.end
 }
 
-do
-{
+do {
     // snippet.EDDSA
     // Initialize an EdDSA key with public PEM
     let publicKey = try EdDSA.PublicKey(x: "...", curve: .ed25519)
 
     // Initialize an EdDSA key with private PEM
-    let privateKey = try EdDSA.PrivateKey(x: "...", d: "...", curve: .ed25519)
```

Comment:
Do we not need this in most cases?

Reply:
The init is still there, it just was renamed to `init(d:, curve:)` because the `x` represents the public key and isn't needed there